showkase/src/main/java/com/airbnb/android/showkase/ui/CommonComponents.kt

```diff
@@ -57,27 +60,34 @@ internal fun ComponentCardTitle(componentName: String) {
 @Composable
 internal fun ComponentCard(
     metadata: ShowkaseBrowserComponent,
-    onClick: (() -> Unit)? = null
+    onClick: (() -> Unit)? = null,
+    darkMode: Boolean = false,
 ) {
     val composableModifier = Modifier.generateComposableModifier(metadata)
     val composableContainerModifier = Modifier.generateContainerModifier(onClick)
-    Card {
-        Box {
-            Column(modifier = composableModifier) {
-                metadata.component()
+    MaterialTheme(
```

Comment:
Ideally this was handled at the root screen itself but since I just realized Showkase wasn't really using the theme at all, I'm limit the scope and restrict it to just the dark mode use case.